### ISSUE TITLE
Please consider a less generic name

### ISSUE THREAD

**Issue by unixorn** [trobjo/zsh-plugin-manager#1]
I'd like to add this to [awesome-zsh-plugins](https://github.com/unixorn/awesome-zsh-plugins), but the name is very generic. Please consider renaming it to something descriptive, or even just name it after your github username.

Thanks!

**Comment by trobjo** [comment]
Absolutely! I do not know what to call it yet, but I will think of a name. And I am open to suggestions.

**Comment by unixorn** [comment]
Maybe minimal-manager because it's so nice and short?

**Comment by unixorn** [comment]
Or maybe mini-roster because it controls your list of plugins/themes?
